mne/io/eyelink/eyelink.py

```diff
@@ -284,6 +287,45 @@ def _find_overlaps(df, max_time=0.05):
     return ovrlp.drop(columns=tmp_cols).reset_index(drop=True)
 
 
+@fill_doc
+def read_eyelink_calibration(
+    filename, screen_size=None, screen_distance=None, screen_resolution=None
+):
+    """Return info on calibrations collected in an eyelink file.
+
+    Parameters
+    ----------
+    filename : str
+        Path to the eyelink file (.asc).
```

Comment:
This argument is usually called `fname` in the other IO functions.

Reply:
> A couple of minor comments after gazing through the documentation, which can be applied to other docstrings too. Looks great!

Thanks! I've committed your suggestions and will take another pass at the docstring to make similar changes in places that could benefit from it.

> Great @scott-huberty ! I'll give it a try no my EyeLink data as well and report back! 🙂

Sounds good! 🙂

Reply:
changed `filename` to `fname` in commit [87320fb](https://github.com/mne-tools/mne-python/pull/11719/commits/87320fb0e2e7a3df63a0efef2193df171a75970c)